src/main/java/ch/njol/skript/aliases/ItemType.java

```diff
@@ -1288,6 +1288,13 @@ public Material getMaterial() {
 		return data.getType();
 	}
 
+	public ItemStack getItemStack() {
```

Comment:
Why are you adding this method to the ItemType? ItemType has a method called `getRandom()` which generates a random ItemStack based off the ItemType details. The method name is misleading yes, but if we're doing this, this need to deprecate the old method. Please do that in another pull request or use the `getRandom()` method.